src/main/java/com/suse/saltstack/netapi/config/ClientConfig.java

```diff
@@ -35,6 +35,13 @@
     public static final Key<String> PROXY_PASSWORD = new Key<>();
 
     /**
+     * Maximum websocket message length in characters. The default value corresponds to
+     * 10 MB, assuming that one character takes up 2 bytes. This limit can be disabled
+     * by setting a value <= 0.
+     */
+    public static final Key<Integer> WEBSOCKET_MAX_MESSAGE_LENGTH = new Key<>(5242880);
```

Comment:
My calculation is correct I guess, note that the limit is defined in **characters** :wink: Changed the notation to hexadecimal though with a8a5cb9382171b77f9a7d08519d59bcd10fecb24.